diff --git a/tests/test_ansi2html.py b/tests/test_ansi2html.py
index 04b5d22..67086f9 100644
--- a/tests/test_ansi2html.py
+++ b/tests/test_ansi2html.py
@@ -499,5 +499,5 @@ class TestAnsi2HTML:
         assert process.returncode == 0

     def test_command_module(self) -> None:
-        result = run(["python3", "-m", "ansi2html", "--version"], check=True)
+        result = run(["python", "-m", "ansi2html", "--version"], check=True)
         assert result.returncode == 0
